fastestimator/trace/trace.py

```diff
@@ -214,25 +215,43 @@ class EvalEssential(Trace):
     Args:
         monitor_names: Any keys which should be collected over the course of an eval epoch.
     """
+
     def __init__(self, monitor_names: Set[str]) -> None:
-        super().__init__(mode="eval", inputs=monitor_names)
+        super().__init__(mode="eval", inputs=monitor_names, outputs=["steps/sec"])
+        self.elapse_times = []
+        self.eval_print = None
+        self.step_start = None
         self.eval_results = defaultdict(lambda: defaultdict(list))
 
     def on_epoch_begin(self, data: Data) -> None:
         self.eval_results = defaultdict(lambda: defaultdict(list))
+        self.eval_step = 0
+        self.elapse_times = []
+
+        if self.system.log_steps:
+            self.step_start = time.perf_counter()
```

Comment:
this, as well as resetting self.eval_step to 0 should be done on_epoch_begin so that perf counting in higher epochs isn't messed up initially